boards/DIM/Src/App/states/App_DriveState.c

```diff
@@ -148,7 +139,7 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     {
         App_SevenSegDisplays_SetGroupL(seven_seg_displays, speed_kph);
         App_SevenSegDisplays_SetGroupM(seven_seg_displays, instant_power);
-        App_SevenSegDisplays_SetGroupR(seven_seg_displays, avg_power);
+        App_SevenSegDisplays_SetGroupR(seven_seg_displays, min_cell_voltage);
```

Comment:
@forbesashli should we put avg power on another 7seg? I think speed, instantaneous power and min cell voltage are good for now right?

Reply:
i think this is also a good choice, due to just needing to see things while driving for testing purposes, i think averaged power is not super useful to most drivers for a while